Gruntfile.js

```diff
@@ -18,6 +26,21 @@ module.exports = function(grunt) {
         stripBanners: true,
         banner: '/* <%= pkg.name %> <%= grunt.template.today("yyyy-mm-dd") %> */',
       },
+      static_mappings: {
+        files: [
+          // note:  also need index.html, static/css and static/images files
+          {src: 'server-bfe.js', dest: 'builds/server-bfe.js'},
+          {src: 'static/js/config.js', dest: 'builds/config.js'},
+          {src: 'static/js/jsonld-vis.js', dest: 'builds/jsonld-vis.js'},
```

Comment:
I made ticket #30 re: running it entirely from a builds (or dist) directory, #28 about using webpack instead of grunt.  I will add to #28 re: redoing directory organization.   

It seems better, to me, to get some basic integration tests written before doing any of that as it will help us know if we've done it correctly.

Reply:
👍